worldedit-bukkit/src/main/java/com/sk89q/bukkit/util/CommandRegistration.java

```diff
@@ -97,7 +98,11 @@ public CommandMap getCommandMap() {
         if (commandMap == null) {
             Bukkit.getServer().getLogger().severe(plugin.getDescription().getName()
                 + ": Could not retrieve server CommandMap, using fallback instead!");
-            fallbackCommands = commandMap = new SimpleCommandMap(Bukkit.getServer());
+            try {
+                fallbackCommands = commandMap = SimpleCommandMap.class.getConstructor(Server.class).newInstance(Bukkit.getServer());
+            } catch (Throwable e) {
```

Comment:
I don't like catching this without logging. Why are we doing this new call anyways? Is this Spigot <-> Paper compat? Can it be documented if so?

Reply:
it's for Spigot/Paper compat yes, afaict this will never actually be called as the above conditions will be met- it's more to just retain this compatibility for whatever niche server does it.

in a Paper-only future this whole thing can just be the simple call above

Reply:
I still think we should log, even in a fallback. Not logging in these situations has caused issues before when we needed to debug them.

Reply:
i've dropped the Paper version here given Paper will never need a fallback anyway, and just opted to throw if this fails. It's a non-recoverable position anyway afaik